app/experimenter/nimbus-ui/src/components/App/index.test.tsx

```diff
@@ -51,15 +51,15 @@ describe("App", () => {
       const {
         history: { navigate },
       } = renderWithRouter(
-        <MockedCache mocks={[mock]}>
+        <MockedCache mocks={[mock, mock]}>
```

Comment:
pourquoi two of the same mocks?

Reply:
The same reason as the other. This hits `/my-special-slug/` (summary page) first and then navigates to `/edit`.